src/qute/languageServer/client.ts

```diff
@@ -100,14 +100,15 @@ export function connectToQuteLS(context: ExtensionContext, api: JavaExtensionAPI
           if (window.activeTextEditor?.document === document) {
             await synchronizeQuteValidationButton(window.activeTextEditor);
           }
+          // Display the experimental Qute validation pop-up if it hasn't been displayed
+          if (!hasShownQuteValidationPopUp(context)) {
```

Comment:
Can we do something like :

```
  const quteSupported = document.languageId.includes('qute');
  if (quteSupported) {
    showQuteValidationPopUp(context);
  }
```
Without this, the popup would come up for even Java source files.

Reply:
It seems both checks on gh actions have passed finally?
<img width="916" alt="image" src="https://user-images.githubusercontent.com/26389510/162812791-60b86bbe-56fa-495d-bc85-0ba6c5ae0d4f.png">
Don't know if this will be consistent.

Reply:
Could just be luck of the draw. Either way, would be good to investigate those tests eventually.